src/components/shop/shop-menu-screen.js

```diff
@@ -19,6 +19,8 @@ export class ShopMenu extends Screen {
     }
 
     create() {
+        this.transientData.shopTitle = this.config.title?.text.toLowerCase() || "shop";
```

Comment:
I'm a bit confused by this, title could be set to anything, are you after scene.key instead?

Reply:
yeah wasn't 100% sure what to pick for this, I did deliberately choose this instead of the scene key but who knows

Reply:
Maybe you need to take the start of the scene key eg. "someshop-menu" becomes "someshop"

Reply:
sounds good